dip-template/nodes/dip-consumer/src/service.rs

```diff
@@ -278,42 +297,12 @@ async fn start_node_impl(
 			transaction_pool,
 			sync_service.clone(),
 			params.keystore_container.keystore(),
-			force_authoring,
```

Comment:
So there's not `--force-authoring` flag anymore?

Reply:
Yes, the `force-authoring` flag has been removed